src/scripts/config.js

```diff
@@ -268,13 +268,11 @@ async function processModule(filePath, cachedIds, registry, ids, envs) {
   let prependContent = `window.modules["${moduleId}"] = [function(require,module,exports){`;
   let appendContent = '';
 
-  const fileName = `${moduleId}.js`;
-
   // compile vue to JS before creating the magic string
   if (isVue) {
     try {
       const vueSFC = await vueSfcHandler({
-        filename: fileName,
+        filename: filePath,
```

Comment:
was getting an error processing assets in vue files like here, which filePath resolves. but maybe then breaks your use case?

```
<style lang="scss">
@use "../../../global/scss/utility/_visibility.scss" as visibility;
```